### ISSUE TITLE
Definition for rule 'quote-probs' was not found 

### ISSUE THREAD

**Issue by cytopia** [eslint/eslint#4557]
I am using the following `quote-probs` rule:

```json
    "quote-probs": [2, "as-needed", { "keywords": true, "unnecessary": false }],
```

When I run `eslint` it tells me that the rule was not found:
```bash
$ eslint .
 1:1  error  Definition for rule 'quote-probs' was not found  quote-probs

✖ 1 problem (1 error, 0 warnings)
```

Anything that I am missing?

**Comment by lo1tuma** [comment]
The rule is called `quote-props` not `quote-probs`.

**Comment by cytopia** [comment]
My bad. Thanks!
